console-extensions.json

```diff
@@ -345,5 +387,67 @@
         "HIDE_STATIC_PIPELINE_PLUGIN_PIPELINERUN_DETAIL_TASKRUNS_TAB"
       ]
     }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "path": ["/dev-pipelines/ns/:ns", "/dev-pipelines/all-namespaces"],
+      "exact": false,
+      "component": {
+        "$codeRef": "pipelinesList.PipelinesTabbedPage"
+      }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINE_NAV_OPTION"]
+    }
+  },
+  {
+    "type": "console.page/route",
+    "properties": {
+      "path": ["/pipelines/ns/:ns", "/pipelines/all-namespaces"],
+      "exact": false,
+      "component": {
+        "$codeRef": "pipelinesList.PipelinesTabbedPage"
+      }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINE_NAV_OPTION"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "pipelines",
+      "perspective": "admin",
+      "section": "pipelines",
+      "name": "%Pipelines%",
+      "href": "/pipelines",
+      "namespaced": true,
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-pipelines"
+      }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINE_NAV_OPTION"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "pipelines",
+      "perspective": "dev",
+      "section": "resources",
+      "insertAfter": "builds",
+      "name": "%Pipelines%",
+      "href": "/dev-pipelines",
+      "namespaced": true,
+      "dataAttributes": {
+        "data-quickstart-id": "qs-nav-pipelines",
+        "data-test-id": "pipeline-header"
+      }
+    },
+    "flags": {
+      "required": ["HIDE_STATIC_PIPELINE_PLUGIN_PIPELINE_NAV_OPTION"]
+    }
```

Comment:
https://github.com/openshift-pipelines/console-plugin/assets/102503482/5728d8da-e1f9-439e-89e5-e4cc5ad670c8

I have changed Repositories tab name to Repositories-1 for testing in dynamic plugin, I am able to see changes in Admin view but in Developer perspective I am not able to see changes. May be it is not getting replaced. 

I think the issue - https://github.com/openshift-pipelines/console-plugin/pull/82/files#r1537071748 also because of this

Reply:
This PR will fix it https://github.com/openshift/console/pull/13701. A flag is not set in the static plugin.